app/controllers/cart_items_controller.rb

```diff
@@ -10,10 +10,19 @@ def create
     redirect_to items_path
   end
 
+  def update
+    if params[:quantity] == "0"
+      destroy
+    else
+      @cart.contents.select { |item, quantity| item == params[:id] }.map { |item, quantity| @cart.contents[item] = params[:quantity].to_i }
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.<br>Line is too long. [139/80]